templates/miq-template.yaml

```diff
@@ -52,6 +52,24 @@ objects:
     admin-password: "${APPLICATION_ADMIN_PASSWORD}"
     database-url: postgresql://${DATABASE_USER}:${DATABASE_PASSWORD}@${DATABASE_SERVICE_NAME}/${DATABASE_NAME}?encoding=utf8&pool=5&wait_timeout=5
     v2-key: "${V2_KEY}"
+- apiVersion: v1
+  data:
+    miq_logs.conf: |
+      /var/www/miq/vmdb/log/*.log /var/www/miq/vmdb/log/apache/*.log {
+        daily
+        dateext
+        missingok
+        rotate 14
+        notifempty
+        compress
+        copytruncate
+        lastaction
+          /sbin/service httpd reload > /dev/null 2>&1 || true
+        endscript
+      }
+  kind: ConfigMap
```

Comment:
Can you move the kind and metadata tags up under `apiVersion` for consistency with the rest of the file?